README.md

```diff
@@ -2,23 +2,40 @@
 
 This is a tiny (~1.08k brotli'd) library that mostly mirrors the IndexedDB API, but with small improvements that make a big difference to usability.
 
-1. [Installation](#installation)
-1. [Changes](#changes)
-1. [API](#api)
-   1. [`openDB`](#opendb)
-   1. [`deleteDB`](#deletedb)
-   1. [`unwrap`](#unwrap)
-   1. [`wrap`](#wrap)
-   1. [General enhancements](#general-enhancements)
-   1. [`IDBDatabase` enhancements](#idbdatabase-enhancements)
-   1. [`IDBTransaction` enhancements](#idbtransaction-enhancements)
-   1. [`IDBCursor` enhancements](#idbcursor-enhancements)
-   1. [Async iterators](#async-iterators)
-1. [Examples](#examples)
-1. [TypeScript](#typescript)
+- [IndexedDB with usability.](#indexeddb-with-usability)
```

Comment:
Why is all this changing?

Reply:
@jakearchibald Sorry about this.

I notice it was modified and did not notice it changed from `1.` to `-`. It must be the VS.code extensions, I thought it was just adding the additional headers for 

```
## Directly In a browser

### Using the modules method directly via unpkg:
```

I will revert those

Reply:
I reverted the changes to the TOC. Sorry for the beginner's mistake. I turn off the formatting extensions on this project on my workspace going forward